xarray/core/dataarray.py

```diff
@@ -18,7 +18,9 @@
 from .formatting import format_item
 from .options import OPTIONS
 from .pycompat import OrderedDict, basestring, iteritems, range, zip
-from .utils import decode_numpy_dict_values, ensure_us_time_resolution
+from .utils import (
+    combine_pos_and_kw_args, decode_numpy_dict_values,
```

Comment:
While we're at it, maybe we should rename `combine_pos_and_kw_args` to something clearer, maybe `either_dict_or_kwargs`?